src/extension.ts

```diff
@@ -37,7 +37,7 @@ function setupLSP() {
         // Options to control the language client
         const clientOptions: LanguageClientOptions = {
             // Register the server for racket documents
-            documentSelector: [{ scheme: "file", language: "racket" }],
+            documentSelector: [{ scheme: "file", language: "fracas" }],
```

Comment:
I wonder if this is all that's needed for it to work with both fracas and racket?
```            documentSelector: [{ scheme: "file", language: "racket" }, { scheme: "file", language: "fracas" }],
```

Reply:
i'll give it a shot now!

Reply:
It worked! Thanks for the tip